src/utils/avatar.js

```diff
@@ -398,10 +398,10 @@ export class UploadedAvatarURL extends AvatarURL {
          */
       result = new URL(this._standardUrl);
 
-      const match = new RegExp(/(\w+)\.png/).exec(result.pathname);
-      if (match !== null) {
-        result.pathname = result.pathname.replace(match[0], `${match[1]}-medium.png`);
-      }
+      result.pathname = result.pathname.replace(
+        new RegExp(/(\w+)\.png/),
+        (_, p1) => `${p1}-medium.png`,
```

Comment:
Ah I see, the needs of writing this `replace` call when the replacement is still dependent on the match make this more complicated than I'd quite thought through when I made that suggestion :sweat_smile: 

This revision looks good, though. I'll go ahead and merge.